ionic-angular-template/src/auth/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm, FormGroup, FormBuilder, Validators, FormControl } from '@angular/forms';
+import { AuthService } from '../../services/auth.service';
+import { Router } from '@angular/router';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.component.html',
+  styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit {
+  public signInForm: FormGroup;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private alertCtrl: AlertController,
+    private formBuilder: FormBuilder
+  ) { }
+
+  ngOnInit() {
+    this.signInForm = this.formBuilder.group({
+      email: new FormControl('', Validators.compose([Validators.required, Validators.email])),
+      password: new FormControl('', Validators.compose([Validators.required, Validators.minLength(6)]))
+    });
+  }
+
+  get email() { 
+    return this.signInForm.get('email'); 
+  }
+
+  get password() { 
+    return this.signInForm.get('password'); 
+  }
+
+  async signIn(email: string, password: string) {
```

Comment:
You can get the values of `email` and `password` from the getters. This was marked as resolved, but I can still see that is not changed.